deploy/assisted-service.yaml

```diff
@@ -26,6 +26,15 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 8090
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8090
+            initialDelaySeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: 8090
```

Comment:
Since you expect the container to take up to 30s for starting up, you can add the `initialDelaySeconds` here as well.

Reply:
So just to understand, why is it important? The pod won't be ready until the readinessProb succeeses.
So basically, if I understood correctly, the only thing this change will do is to save some http check for k8s, is that important? Or just a matter of clearness?
@apahim

Reply:
yep, it just saves some probes that are doomed to fail anyway. Not that important.